clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -5274,11 +5274,11 @@ def AtomicCmpXchg : CIR_Op<"atomic.cmp_xchg",
 def SignBitOp : CIR_Op<"signbit", [Pure]> {
   let summary = "Checks the sign of a floating-point number";
   let description = [{
-    It returns a non-zero value (true) if the number is negative
-    and zero (false) if the number is positive or zero.
+    It returns true if the number is negative
+    and false if the number is positive or zero.
```

Comment:
A lot of other floating-point values also set the sign bit such as negative infinity, negative zero, and some NaN values, maybe the doc could be more precise:

```suggestion
    It returns whether the sign bit (i.e. the highest bit) of the input operand
    is set.
```